api/app/signals/apps/questionnaires/fieldtypes/selected_object.py

```diff
@@ -0,0 +1,172 @@
+                                'type': 'object',
+                                'properties': {
+                                    'address': {
+                                        'type': 'object',
+                                        'properties': {
+                                            'postcode': {
+                                                'type': 'string',
+                                                'pattern': '^[1-9][0-9]{3} ?[A-Za-z]{2}$',
+                                            },
+                                            'huisnummer': {
+                                                'anyOf': [
+                                                    {
+                                                        'type': 'integer',
+                                                        'minimum': 0
+                                                    },
+                                                    {
+                                                        'type': 'string'
+                                                    }
+                                                ]
+                                            },
+                                            'woonplaats': {
+                                                'type': 'string',
+                                            },
+                                            'openbare_ruimte': {
+                                                'type': 'string',
+                                            }
+                                        },
+                                    },
+                                    'coordinates': {
+                                        'type': 'object',
+                                        'required': [
+                                            'lat',
+                                            'lng',
+                                        ],
+                                        'properties': {
+                                            'lat': {
+                                                'type': 'number',
+                                                'minimum': -90,
+                                                'maximum': 90
+                                            },
+                                            'lng': {
+                                                'type': 'number',
+                                                'minimum': -180,
+                                                'maximum': 180
+                                            }
+                                        }
+                                    }
+                                },
+                            },
+                            'status': {
+                                'type': 'string'
+                            },
+                        },
+                    },
+                ],
+            },
+            "category_url": {
```

Comment:
What is `category_url` used for? For the "old" system we had planned to filter extra properties on category permissions. Is that the plan here and if so why only for object selection and not other field types?

Reply:
Started with the FE implementation of storing a selected object in the extra_properties. This is something we need to discuss and agree upon what is still relevant.